pupa/importers/base.py

```diff
@@ -191,18 +191,24 @@ def import_data(self, data_items):
         # keep counts of all actions
         record = {
             'insert': 0, 'update': 0, 'noop': 0,
-            'start': datetime.datetime.utcnow(),
+            'start': utcnow(),
+            'records': {
+                'insert': [],
+                'update': [],
+                'noop': [],
+            }
         }
 
         for json_id, data in self._prepare_imports(data_items):
             obj_id, what = self.import_item(data)
             self.json_to_db_id[json_id] = obj_id
+            record['records'][what].append(obj_id)
```

Comment:
this is so we can get all the IDs of all the objects we've fiddled with in the random stages